client/src/components/web-extension/ImportWallet/FieldKeyphrase.jsx

```diff
@@ -3,45 +3,33 @@ import PropTypes from 'prop-types';
 import { Field, useFormikContext } from 'formik';
 
 const FieldKeyphrase = ({ totalWords }) => {
-	const	leftKeys = Object.keys(new Array(totalWords/2).fill(''));
-	const rightKeys = Object.keys(new Array(totalWords/2).fill(''));
+	const keys = Object.keys(new Array(totalWords).fill(''));
 	const { setFieldValue } = useFormikContext();
 
 	const pasteEventHandler = useCallback((event) => {
-		console.log('>> PASTE: ', event.clipboardData.getData('text'));
+		event.preventDefault();
+		
 		const pasteData = event.clipboardData.getData('text');
-		console.log(`🚀 ~ pasteEventHandler ~ pasteData`, pasteData)
 		setFieldValue('keyphrase', pasteData.split(' '));
```

Comment:
@hoailinh1210 

When pasting an invalid keyphrase, or short one (which is less than 12 words), there's a warning in devTool like this:
![image](https://user-images.githubusercontent.com/610234/188278679-f4281c74-43b0-45f6-b833-407307517c3e.png)

That was because of my testing with the above line:  `pasteData.split(' ')`  , we tried to set new value into Form without checking how many words we had. Initially this field requires 12 words in an array,  when pasting less than 12 words, that's invalid from Formik point of view, that's why we have that warning.

Can you try to make sure we set value with exact 12 slot array? 
E:g: Pasting `1 2 3 4 5 6 7 8 9 10`, the array should become `[1,2,3,4,5,6,7,8,9,10,'','']`, we add missing blank value to make this array having 12 slots.

Let me know if this is unclear,

Cheers